src/main/java/core/basesyntax/UserService.java

```diff
@@ -2,10 +2,15 @@
 
 public class UserService {
```

Comment:
```suggestion
public class UserService {
       private PasswordValidator validator = new PasswordValidator();
```

Reply:
done, 
but without "private"
travis don't work

Reply:
it works, i told you to declare it as a class variable not method variable